src/cf-forms/src/molecules/form-fields.less

```diff
@@ -8,19 +8,22 @@
         margin-top: unit( 5px / @base-font-size-px, em );
     }
 
+    /*
+    We need to turn off autoprefixing for the inline-grid because
+    older IE does not handle an inline-grid like other browsers,
+    leading to an extremely narrow column of text for the label.
+     */
```

Comment:
It's probably worth mentioning why this has to be in its own block.

Reply:
Done